pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdClusters.java

```diff
@@ -83,11 +83,67 @@ void run() throws PulsarAdminException {
         @Parameter(names = "--proxy-protocol", description = "protocol to decide type of proxy routing eg: SNI", required = false)
         private ProxyProtocol proxyProtocol;
 
+        @Parameter(names = "--tls-enable", description = "Enable tls connection", required = false)
+        private boolean brokerClientTlsEnabled = false;
+
+        @Parameter(names = "--tls-allow-insecure", description = "Allow insecure tls connection", required = false)
+        private boolean tlsAllowInsecureConnection = false;
+
+        @Parameter(names = "--tls-enable-keystore", description = "Whether use KeyStore type to authenticate", required = false)
+        private boolean brokerClientTlsEnabledWithKeyStore = false;
+
+        @Parameter(names = "--tls-trust-store-type", description = "TLS TrustStore type configuration for internal client eg: JKS", required = false)
+        private String brokerClientTlsTrustStoreType = "JKS";
+
+        @Parameter(names = "--tls-trust-store", description = "TLS TrustStore path for internal client", required = false)
+        private String brokerClientTlsTrustStore;
+
+        @Parameter(names = "--tls-trust-store-pwd", description = "TLS TrustStore password for internal client", required = false)
+        private String brokerClientTlsTrustStorePassword;
+
+        @Parameter(names = "--tls-trust-certs-filepath", description = "path for the trusted TLS certificate file", required = false)
+        private String brokerClientTrustCertsFilePath;
+
```

Comment:
Maybe we could add a new base class for Create and Update command, there are many same configurations.